idb/postgres/postgres_integration_test.go

```diff
@@ -28,6 +38,9 @@ func getAccounting(round uint64, cache map[uint64]bool) *accounting.State {
 
 // setupPostgres starts a gnomock postgres DB then returns the connection string and a shutdown function.
 func setupPostgres(t *testing.T) (*sql.DB, string, func()) {
+	if !enableDocker {
```

Comment:
These can't be disabled by default, it needs to be an action people take so that they aren't forgotten about. You could use the "short" option and have them disabled `go test -short`, or use something like "go test `go list ./... | grep -v integration_test`"